app/javascript/components/student/tracks-list/Search.jsx

```diff
@@ -12,6 +12,7 @@ export function Search({ dispatch }) {
     <input
       type="text"
```

Comment:
Consider `type="search"`

The form should have [`role="search"`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Search_role)

Reply:
https://github.com/exercism/v3-website/issues/179